modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -30,13 +31,17 @@ import (
 	"go.uber.org/zap"
 )
 
-func (fsrv *FileServer) directoryListing(entries []fs.DirEntry, canGoUp bool, root, urlPath string, repl *caddy.Replacer) browseTemplateContext {
+func (fsrv *FileServer) directoryListing(ctx context.Context, entries []fs.DirEntry, canGoUp bool, root, urlPath string, repl *caddy.Replacer) browseTemplateContext {
 	filesToHide := fsrv.transformHidePaths(repl)
 
 	var dirCount, fileCount int
-	fileInfos := []fileInfo{}
+	var fileInfos []fileInfo
```

Comment:
Huh. I thought I'd save an allocation if the dir is empty, but linter thinks otherwise.